pkgs/applications/misc/nwg-look/default.nix

```diff
@@ -0,0 +1,69 @@
+{ stdenv
+, fetchgit
+, pkgs
+, lib
+, buildGoModule
+, go
+, glib
+, pkg-config
+, cairo
+, gtk3
+, xcur2png
+, libX11
+, zlib
+}:
+
+buildGoModule rec {
+  pname = "nwg-look";
+  version = "0.2.4";
+
+  src = fetchgit {
+    url = "https://github.com/nwg-piotr/nwg-look";
+    rev = "f0f9f7278d7cbaa85862ad9a0ab42886c5938d61";
+    hash = "sha256-wUI58qYkVYgES87HQ4octciDlOJ10oJldbUkFgxRUd4=";
+  };
+
+  vendorHash = "sha256-dev+TV6FITd29EfknwHDNI0gLao7gsC95Mg+3qQs93E=";
+
+  patches = [ ./fix-paths.patch ];
```

Comment:
You can do such substitutions directly on fix-paths.patch.

Reply:
Could you explain more please, sorry I am quite new.

Reply:
Do you mean something like this:
```bash
sed -i 's|@out@|'"''${out}"'|g' main.go tools.go
```

Reply:
Even better: you can use substituteAll in the patch, obviating the postPatch.

Look at pkgs/applications/misc/valentina/default.nix as an example.

Reply:
I do not believe that this would be very useful in this case as I have not found a way to inherit the correct variables to make the substitutions, it may be obvious but I have not found it.

Reply:
`inherit X` is basically a syntatic sugar for `X = X`. It is no too complicated...